src/jsi/jsi-inl.h

```diff
@@ -30,7 +30,7 @@ inline Value toValue(Runtime& runtime, const char* str) {
   return String::createFromAscii(runtime, str);
 }
 inline Value toValue(Runtime& runtime, const std::string& str) {
-  return String::createFromAscii(runtime, str);
+  return String::createFromUtf8(runtime, str);
```

Comment:
>createFromUtf8 [](start = 17, length = 14)

Thanks!

Reply:
Note that all files under src/jsi are simply copies from Facebook's Hermes project. They need to stay in sync with the upstream repo so I won't make any changes there.